docs/asciidoc/tcp-client.adoc

```diff
@@ -310,6 +310,12 @@ include::{examplesdir}/proxy/Application.java[lines=18..38]
 ----
 ====
 
+NOTE: Netty's HTTP proxy support always uses `CONNECT` method in order to establish a tunnel to the specified proxy regardless of the scheme that is used `http` or `https`.
```

Comment:
unless I missed something this is a copy-paste of the faq entry. consider linking to the faq entry or using a [tagged region](https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/) to directly import the relevant part(s) from the `faq.adoc` source

Reply:
ok